jarbas/api/tests/test_reimbursement_view.py

```diff
@@ -110,6 +111,7 @@ def _count_results(self, url):
         return len(content.get('results', 0))
 
 
+@freeze_time('1970-01-01 00:00:00')
```

Comment:
I've never noticed it. But this is only the second time ever that I use that, so I might not be the  best person to have a say on that ; )